template/node/src/chain_spec.rs

```diff
@@ -165,7 +165,9 @@ fn testnet_genesis(
 			accounts: {
 				let mut map = BTreeMap::new();
 				map.insert(
-					H160::from_str("6be02d1d3665660d22ff9624b7be0551ee1ac91b")
+					// H160 address of Alice dev account
+					// Derived from SS58 (42 prefix) address 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+					H160::from_str("57d213d0927ccc7596044c6ba013dd05522aacba")
```

Comment:
NOTE: this may have implications in testing. I am using the README's suggested H160 encoded address for Alice. Not sure where the previous one comes from... 🤔

Reply:
`6be02...` is an account that the Moonbeam team used in testing, as sudo, and in all kinds of places for development. We call him "Gerald". Recently we added ethereum-style account derivation and proper dev accounts, so Gerald retired.

He probably shouldn't be used in this repo.

Reply:
Glad to know the origin. Sad to see him go.
> Gerald is dead. Long live Gerald!